docs/code-search/code-navigation/how-to/adding_lsif_to_workflows.mdx

```diff
@@ -64,26 +41,21 @@ If your repository contains multiple code projects in different folders, your CI
 
 ```yaml
 jobs:
-  scip-go:
-    # Prevent forks of this repo from uploading SCIP indexes
+  lsif-go:
```

Comment:
Varun mentioned this in other places but this is a bad merge, I recently updated this content to be correct for  `scip-go`